controllers/gatekeeper_controller.go

```diff
@@ -264,6 +286,41 @@ func (r *GatekeeperReconciler) SetupWithManager(mgr ctrl.Manager, fromCPSMgrSour
 		Complete(r)
 }
 
+func (r *GatekeeperReconciler) initConfig(ctx context.Context, gatekeeper *operatorv1alpha1.Gatekeeper) error {
+	config := &v1alpha1.Config{}
+
+	err := r.Get(ctx, types.NamespacedName{
+		Namespace: r.Namespace,
+		Name:      "config",
+	}, config)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			err = createDefaultConfig(ctx, r.Client, r.Namespace, gatekeeper, r.Scheme)
+			if err != nil {
+				return err
+			}
+
+			r.Log.Info("The Gatekeeper's Config resource is just created")
+
+			return nil
+		}
+
+		return err
+	}
+
+	// Add the default exempt namespace list
+	// disableDefaultMatches = true, only append gatekeeper.config.matches and
+	// the default exempt namespaces will not be appended
+	err = setExemptNamespaces(ctx, r.Client, config, gatekeeper, r.Scheme, r.Log)
```

Comment:
Optional: This might be a good candidate to convert to a method on the reconciler struct since you pass in three struct members.

Reply:
it is used in both config_controller and gatekeeper_controller.